Gemfile.lock

```diff
@@ -207,6 +205,13 @@ GEM
       addressable (>= 2.8)
     jsonapi-serializer (2.2.0)
       activesupport (>= 4.2)
+    karafka-core (2.1.1)
+      concurrent-ruby (>= 1.1)
+      karafka-rdkafka (>= 0.13.1, < 0.14.0)
```

Comment:
Hmm, this doesn't sound convincing 😞 seems like karafka has its own fork of rdkafka. One of the reasons to use waterdrop was that it will use the same rdkafka gem for producing and consuming messages as well, but it seems that it is not true. 

I am not saying that we should throw away your PR, because eventually I want to migrate us towards a single producer/consumer, but as long as we don't have an alternative for racecar, we should probably stick with a single library. 

Do you think you could stash this PR somewhere and for now just revive mine as a substitution for the ruby-kafka producer we are using at the moment?